.github/workflows/unittests.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-13, windows-latest]
```

Comment:
Note: we should probably also upgrade to `macos-latest`; that label currently references MacOS 14, so our `macos-13` entry is out-of-date.